api/lib/models/event/event-data-mapper-impl.ts

```diff
@@ -84,10 +84,11 @@ export class EventDataMapperImpl extends GenericDataMapper implements IDataMappe
       .order('event_start_time', true)
       .join('LOCATIONS', 'location', 'event_location=location.uid')
       .join('HACKATHON', 'h', 'h.uid=event.hackathon and h.active=true')
-      .toString()
+      .toParam();
+    query.text = query.text
       .concat(';');
     const params = [];
-    return from(this.sql.query<Event>(query, params, { cache: true }))
+    return from(this.sql.query<Event>(query.text, params, { cache: true }))
```

Comment:
Why not use query.values here? seems important?

Reply:
I'm not entirely sure because I didn't originally write this but I can change it to query.values like the others